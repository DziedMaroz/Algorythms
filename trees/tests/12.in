11
7
12
4
13
2
5
6
3
1
8
9
10
14
