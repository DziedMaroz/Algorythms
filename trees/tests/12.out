11
8
4
2
1
3
5
6
9
10
12
13
14

